keyboards/ducky/one2mini/chconf.h

```diff
@@ -0,0 +1,762 @@
+/*
```

Comment:
This entire file can be replaced with:

```c
/* Copyright 2020 QMK
 *
 * This program is free software: you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation, either version 2 of the License, or
 * (at your option) any later version.
 *
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */

/*
 * This file was auto-generated by:
 *    `qmk chibios-confmigrate -i keyboards/ducky/one2mini/chconf.h -r platforms/chibios/common/configs/chconf.h`
 */

#pragma once

#define CH_DBG_SYSTEM_STATE_CHECK TRUE

#include_next <chconf.h>
```

Reply:
This suggestion breaks the keyboard

Reply:
Please execute the command listed in my suggestion, and work out which parts are needed. Full ChibiOS config files are to be avoided.

Reply:
ok